pages/your_domain/structural_bioinformatics.md

```diff
@@ -1,36 +1,50 @@
 ---
 title: Structural Bioinformatics
-search_exclude: true
-contributors: [<!---REPLACE THIS with comma separated list of contributors--->]
-page_id: struct info
+contributors: [Gerardo Tauriello, Ian Sillitoe, Nicola Bordin, Christine Orengo, Mihaly Varadi, Sameer Velankar, Jiří Černý]
```

Comment:
```suggestion
contributors: [Gerardo Tauriello, Ian Sillitoe, Nicola Bordin, Christine Orengo, Mihaly Varadi, Sameer Velankar, Jiří Černý]
coordinators: [Gerardo Tauriello]
```


We support since shortly also the role of coordinators. These are people that are involved in helping maintaining the page. Would you be interested to be such a coordinator? @gtauriello